pkg/config/parameters.go

```diff
@@ -613,6 +613,10 @@ type Parameters struct {
 	// See: https://github.com/projectcontour/contour/issues/3221
 	DisableAllowChunkedLength bool `yaml:"disableAllowChunkedLength,omitempty"`
 
+	// MergeSlashes toggles Envoy's non-standard merge_slashes path transformation option
+	// which strips duplicate slashes from request URL paths.
+	MergeSlashes bool `yaml:"mergeSlashes,omitempty"`
```

Comment:
Would also switch this one to `DisableMergeSlashes` with a default value of `false`. Otherwise, if this field is not specified in the config file, it's indistinguishable from it being set to `false` which would change the default behavior for users.